src/Apps/Order/Routes/__tests__/Shipping.jest.tsx

```diff
@@ -1091,7 +1101,7 @@ describe("Shipping", () => {
         // expect(screen.getByRole("radio", { name: /White Glove/ })).toBeVisible()
         // expect(screen.getByRole("radio", { name: /Rush/ })).toBeVisible()
         // expect(screen.getByRole("radio", { name: /Premium/ })).toBeVisible()
-
+        triggerLeakyFailure()
```

Comment:
I have a theory about the leak: The [mockCommitMutation](https://github.com/artsy/force/blob/161e26e1bdd2416d83cf7f4bd084c73288bf0c85/src/Apps/Order/Routes/__tests__/Shipping.jest.tsx#L80) was mocked above but we [didn't reset it properly](https://github.com/artsy/force/blob/161e26e1bdd2416d83cf7f4bd084c73288bf0c85/src/Apps/Order/Routes/__tests__/Shipping.jest.tsx#L188) after each test. So when a test fails here, the uncleared mock would leak to the subsequent test(s) and cause it to fail too.

Maybe we should do `jest.resetAllMocks()` instead of `jest.clearAllMocks()`.